signup.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Signup</title>
+    <link rel="stylesheet" href="signupStyle.css" />
+  </head>
+  <body>
+    <div class="signup">
+      <div class="logo">
+        <img class="logo-image" src="image/logo.png" alt="logo" />
+        <a href="/">
+          <img class="pandamarket-logo" src="image/pandamarket.png" alt="pandamarket" />
+        </a>
+      </div>
+      <div class="signup-form">
+        <div class="signup-box">
+          <div class="email">
+            <h3>이메일</h3>
+            <input type="text" placeholder="이메일을 입력해주세요" />
```

Comment:
`input`을 위한 텍스트는 `label` 태그를 사용해주셔야합니다. 
```suggestion
            <label>이메일</label>
            <input type="text" placeholder="이메일을 입력해주세요" />
```